src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.PowerFx
     /// </summary>
     [DebuggerDisplay("{_debugName}")]
     [ThreadSafeImmutable]
-    public class ReadOnlySymbolTable : INameResolver, IGlobalSymbolNameResolver
+    public class ReadOnlySymbolTable : INameResolver, IGlobalSymbolNameResolver, IEnumStore
```

Comment:
>IEnumStore [](http://example.com/codeflow?start=81&length=10)

IEnumStore is conceptually part of the symbol table, just like functions, names, etc.